candies/cli/parsers/standard.py

```diff
@@ -78,6 +78,34 @@ def format_help(self):
     return Help
 
 
+def prefixes(signature: inspect.Signature) -> str:
+    """Constructs a prefixes string for a `signature`.
+
+    For example, if `signature` has `Opt(prefix='/')` and `Opt(prefix='+')`
+    arguments, then the resulting string would be `/+`.
+    """
+
+    def prefix(x: inspect.Parameter) -> str:
+        if isinstance(x.annotation, type):
+            # This allows writing `x: Flag` and expect `--x` to be parsed.
+            return '-'
+        else:
+            return getattr(x.annotation, 'prefix', None)
+
+    def prefix_is_one_character(x: str) -> bool:
+        return x is not None and len(x) == 1
```

Comment:
Do we really need to check for `len(x) == 1`? As far as I understand, if any `len(prefix) > 1`, this method will fail with an exception anyway. If so, I believe we could get rid of `prefix_is_one_character` by calling `getattr(x.annotation, 'prefix', '-')`.

Reply:
Agreed. I've refactored code a bit. Could you please take a look?